projects/11-split.json

```diff
@@ -0,0 +1,29 @@
+{
+    "driver": {
+        "num": 200,
+        "typename": "simpixel"
+    },
+
+    "layout": "strip",
+
+    "aliases": {
+        "bpa": "BiblioPixelAnimations.strip",
+        "Fill": "BiblioPixelAnimations.simple.Fill"
+    },
+
+    "run": {
+        "seconds": 23,
+        "fps": 30
+    },
+
+    "animation": {
+        "typename": "split",
+        "size": 50,
+        "animations": [
+            {"typename": "@Fill", "color": "blue"},
+            "@bpa.Wave",
```

Comment:
How does it know this is bpa strip wave and not some other one? (There's not, I'm just curious... I know this is old).

Reply:
Dunno why I seem to have lost my ability to perform a review, and have to answer each one individually!

The answer is on lines 9-10:

    "aliases": {
        "bpa": "BiblioPixelAnimations.strip",